packages/three-vrm-core/package.json

```diff
@@ -52,7 +52,6 @@
   },
   "devDependencies": {
     "@types/three": "^0.154.0",
-    "lint-staged": "13.1.2",
```

Comment:
Root lint-staged seems to work with per-package "lint-staged" config but currently html/js files aka `lint-examples` is not included. Is this intended?

Reply:
`lint-examples` should also be handled by lint-staged, indeed! Will do in a separated PR.